src/webaudio/WebAudioNodes.ts

```diff
@@ -64,29 +56,49 @@ export class WebAudioNodes extends Filterable
      */
     public context: WebAudioContext;
 
+    /**
+     * Private reference to the script processor node.
+     * @name PIXI.sound.SoundNodes#_script
+     * @type {ScriptProcessorNode}
+     */
+    private _script: ScriptProcessorNode;
+
     constructor(context: WebAudioContext)
     {
         const audioContext: AudioContext = context.audioContext;
 
         const bufferSource: AudioBufferSourceNode = audioContext.createBufferSource();
-        const script: ScriptProcessorNode = audioContext.createScriptProcessor(0);
         const gain: GainNode = audioContext.createGain();
         const analyser: AnalyserNode = audioContext.createAnalyser();
 
         bufferSource.connect(analyser);
         analyser.connect(gain);
         gain.connect(context.destination);
-        script.connect(context.destination);
 
         super(analyser, gain);
 
         this.context = context;
         this.bufferSource = bufferSource;
-        this.script = script;
         this.gain = gain;
         this.analyser = analyser;
     }
 
+    /**
+     * Get the script processor node.
+     * @name PIXI.sound.SoundNodes#script
+     * @type {ScriptProcessorNode}
+     * @readonly
+     */
+    public get script()
+    {
+        if (!this._script)
+        {
+            this._script = this.context.audioContext.createScriptProcessor(0);
```

Comment:
Could you re-add BUFFER_SIZE that was removed in #88? iOS will need to be able to change to something non-zero if users want to use ScriptProcessorNode.

```typescript
    /**	
     * The buffer size for script processor, default is `0` which auto-detects. If you plan to use
     * script node on iOS, you'll need to provide a non-zero amount.
     * @name PIXI.sound.SoundNodes.BUFFER_SIZE	
     * @type {number}	
     * @default 0	
     */	
    public static BUFFER_SIZE: number = 0;
```

Reply:
done!